src/View/Helper/PictureHelper.php

```diff
@@ -69,7 +69,7 @@ public function webp($image, array $widths = [300, 500], array $params = []): ?s
         }
 
         if (!is_a($image, ImageAsset::class)) {
-            throw new \http\Exception\InvalidArgumentException(
```

Comment:
This FCQN was wrong and I replaced it with the one you already had inside your plugin.